portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,13 +39,12 @@
 public class DataServlet extends HttpServlet {
     /*TODO move class into own file*/
     public class CommentMessage {
-    private String sender; 
-    private String message;
-    private String recipient;
-    public CommentMessage(String sndr, String msg){
-        sender = sndr;
-        message = msg;
-    }
+        private String sender; 
+        private String message;
+        public CommentMessage(String sndr, String msg){
```

Comment:
The constructor is good as is, but people usually also just do:

public CommentString(String sender, String message) {
  this.sender = sender;
  this.message = message;
}